sortbootorder.c

```diff
@@ -131,6 +131,8 @@ int main(void) {
 	struct cbfs_handle *bootorder_handle;
 #endif
 
+    lib_get_sysinfo();
```

Comment:
@mwit-3mdeb indentations are not correct

Reply:
Fixed